### DISCUSSION TITLE
Run code about "hmac-sha256" and "traitlets 5.0"

### DISCUSSION THREAD

**Discussion by lwjmpy** [microsoft/vscode-jupyter#11206] [Questions and Answers]
c:\Users\AppData\Local\Programs\Python\Python310\lib\site-packages\traitlets\traitlets.py:2392: FutureWarning: Supporting extra quotes around strings is deprecated in traitlets 5.0. You can use 'hmac-sha256' instead of '"hmac-sha256"' if you require traitlets >=5.
  warn(
c:\Users\AppData\Local\Programs\Python\Python310\lib\site-packages\traitlets\traitlets.py:2346: FutureWarning: Supporting extra quotes around Bytes is deprecated in traitlets 5.0. Use 'c521b58f-81b8-4097-8bb2-60591c60df0d' instead of 'b"c521b58f-81b8-4097-8bb2-60591c60df0d"'.
  warn(
Bad file descriptor (C:\projects\libzmq\src\epoll.cpp:100). 

How to solve this?

**Comment by IanMatthewHuff** [comment]
@lwjmpy Looks to me like the same issue as here:
https://github.com/microsoft/vscode-jupyter/issues/8630

There is some advice in that thread for helping to get things working.
